docs/pages/blog/2020.md

```diff
@@ -0,0 +1,115 @@
+
+## Looking at 2021
+
+2020 was great, 2021 is going to be even more exciting!
+We will continue in the same direction, it's still [**day one**](https://www.sec.gov/Archives/edgar/data/1018724/000119312517120198/d373368dex991.htm). Here is a breakdown of our [roadmap](/discover-more/roadmap/).
+
+### Branding
+
+Up until now, we never had a mindful reflection on what the branding of Material-UI should be. We picked a color from the 2014 Material Design palette, and that's it.
+The material-ui.com marketing website will soon wear a brand new style! We need to create our own identity, distinct from Material Design.
+
+### Material-UI X
+
+We have started to deliver advanced React components in 2020 with the data grid, including a [commercial version](/components/data-grid/#commercial-version) and the beginning of a [date range picker](https://next.material-ui.com/components/date-range-picker/).
+We will double down on these existing components as long as necessary to have them find the market.
+
+By the end of 2021, we aim to have released these components as stable, implement all the [features planned](/components/data-grid/getting-started/#feature-comparison), and at least double the size of the team.
+
+### Material-UI v5
+
+We will release the next major iteration of the library. A highlight of the key improvements coming ✨:
+
+- Polish and promote most of the components that were in the lab in v4 to the core.
+- A new styled engine. Migrate from JSS to emotion (default) and styled-components's `styled` API.
+- Improve customizability. Add new powers to the theme with dynamic color & [variant](https://next.material-ui.com/customization/typography/#adding-amp-disabling-variants) support. Add a new [`sx` prop](https://next.material-ui.com/system/basics/) for quick customizations to all the components. Expose global class names. Deprecate the `makeStyles` and `withStyles` API.
+- Breaking changes on the API to make it more intuitive.
+- Add full support for React strict mode. We recommend to enable it.
+- Improve the performance of the system by a x3-x5 [factor](https://github.com/mui-org/material-ui/issues/21657#issuecomment-707140999).
+- Reduce bundle size: split IE11 into a different bundle [-6kB](https://github.com/mui-org/material-ui/pull/22814#issuecomment-700995216), migrate to emotion [-5kB](https://github.com/mui-org/material-ui/pull/23308#issuecomment-718748835), Popper.js v2 upgrade [-700B](https://github.com/mui-org/material-ui/pull/21761#issuecomment-657135498).
+- Improve the documentation website: search shortcut, page rating, fast material icons copy button, etc.
+
+And [much more](https://github.com/mui-org/material-ui/issues/20012).
+
+### A second theme
+
+While Material Design is a predominant design choice for building rich client-side applications, it's not in the landscape of landing and marketing pages.
```

Comment:
Why not a driver at all? Isn't what Bootstrap was mostly used for initially? I do agree that it might not be the first driver. How about I inverse the order between the two drivers?

Reply:
> Why not a driver at all?

For the reason given.

Reply:
> most Material-UI components have no place on a home page, landing page, or marketing pages.

Are you saying the following components have no place in a landing page:

- Container
- Grid
- Box
- Tooltip
- Accordion
- Breadcrumbs
- Pagination
- Button
- Icons
- Typography
- Radio
- Select
- Input
- Rating

As well as the ones we have an interest in building in the coming months:

- NProgress
- Scrollspy
- Stack
- Carousel

?

Reply:
https://yourlogicalfallacyis.com/strawman

Given the relative simplicity of the majority of elements that make up a home page (most of the ones listed, if you exclude the clutching at straws) will continue to dominate that space.

Optimising the design of a large majority of the components, including the most complex, for use on a home page or landing page makes no sense.

Reply:
https://yourlogicalfallacyis.com/strawman

Given the relative simplicity of the majority of elements that make up a home page (most of the ones listed, if you exclude the clutching at straws) HTML & CSS will continue to dominate that space.

Optimising the design of all of the components, including the most complex, for use on a home page or landing page makes no sense, when only a small minority of the most simple have any reasonable application there, especially given the tiny minority of sites that might use React for their home page in the first place.

If  we want to follow the design trends of the latest startups, we need to look at their applications and dashboards, not their home pages, since it's where the vast majority of our components play.

Reply:
> If we want to follow the design trends of the latest startups, we need to look at their applications and dashboards, not their home pages, since it's where the vast majority of our components play.

Ok, good for me, I expect they share the same design guidelines and components between the two landscape.